WordPress/gutenberg
# Fix link back to editor from revisions screen

## Description
Fix link back to editor from revisions screen.

* Only show the revisions panel if at least 2 revisions exist. The revisions screen is not designed to display a single revision and is broken in this state. This behavior matches the current core behavior.
* Add a gutenberg query var to the revisions link. This helps identify the correct link to use for returning to the editor from the revisions screen.
* Only reset the classic editor link - if the gutenberg query var is present, use the gurenberg link.

Fixes https://github.com/WordPress/gutenberg/issues/2808

## How Has This Been Tested?
In gutenberg:

* Create post with only title. 
* Change title.
* Go to revisions screen.
* Click return to editor link
Confirm: back in gutenberg

* Create a post with classic editor
* repeat previous steps
Confirm: back to editor link goes back to classic editor.

## Checklist:
Actually, we could probably add a test to confirm the revisions link only appears when there at least two revisions. I feel like we had this test before.
